airbyte-integrations/connectors/source-postgres/src/test-integration/java/io/airbyte/integrations/io/airbyte/integration_tests/sources/AbstractPostgresSourceDatatypeTest.java

```diff
@@ -253,9 +253,14 @@ protected void initTests() {
     addDataTypeTestData(
         TestDataHolder.builder()
             .sourceType("jsonb")
-            .airbyteType(JsonSchemaType.STRING)
-            .addInsertValues("null", "'[1, 2, 3]'::jsonb")
-            .addExpectedValues(null, "[1, 2, 3]")
+            .airbyteType(JsonSchemaType.builder(JsonSchemaPrimitive.ARRAY,
+                JsonSchemaPrimitive.OBJECT,
+                JsonSchemaPrimitive.STRING,
+                JsonSchemaPrimitive.BOOLEAN,
+                JsonSchemaPrimitive.NUMBER)
+                .build())
+            .addInsertValues("null", "'10000'::jsonb", "'true'::jsonb")
```

Comment:
Can you test some more nested json objects here?

Reply:
> Can you test some more nested json objects here?

done